pkg/reconciler/chart/component.go

```diff
@@ -15,6 +15,10 @@ type Component struct {
 	configuration map[string]interface{}
 }
 
+func (c *Component) isRepository() bool {
```

Comment:
Naming can be a bit confusing.. `component.isRepository()`  - maybe we could point out whether its an external component coming form a GIT repo, like `isExternalGitComponent` or something.